src/app/[activityId]/page.tsx

```diff
@@ -57,7 +57,7 @@ export default function ActivityDetail() {
   }
 
   return (
-    <div className='w-full px-30 sm:px-24'>
+    <div className='w-full px-30 max-md:px-24'>
```

Comment:
```suggestion
    <div className='w-full md:px-30 px-24'>
```
💬 `max-~` 설정은 되도록 안 써주시는 게 좋을 것 같습니다~